xpdacq/glbl.py

```diff
@@ -137,24 +137,35 @@ class Glbl:
                  'tri_offset': 13, 'v_asym': 0,
                  'alpha': 2.5, 'tmsk': None}
 
-    def __init__(self, frame_acq_time=FRAME_ACQUIRE_TIME):
-        self._frame_acq_time = frame_acq_time
+    def __init__(self, filepath, internal_dict=None):
```

Comment:
please see: 
http://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

It is a convention to avoid confusion on default values.